lighthouse-core/computed/main-thread-tasks.js

```diff
@@ -177,27 +173,35 @@ class MainThreadTasks {
        */
       case 'v8.compile':
       case 'EvaluateScript':
+        if (task.event.args.data) {
+          taskURLs = [task.event.args.data.url].concat(
+            (task.event.args.data.stackTrace || []).map(entry => entry.url));
+        }
+        break;
       case 'FunctionCall':
-        taskURLs = [argsData.url].concat(stackFrameURLs);
+        taskURLs = [task.event.args.data.url];
```

Comment:
notice I removed access to `stackFrame` - I don't think it exists on `FunctionCall.`